trace/dawr.py

```diff
@@ -64,16 +63,35 @@ def run_cmd(self, bin_var):
         return_value = []
         return child, return_value
 
-    def test_read_dawr_v1(self):
+    def run_test(self, cmd):
+        return process.run(cmd, shell=True)
+
+    def perf_cmd(self, perf_record):
+        process.run(perf_record, shell=True, ignore_status=True,
+                    verbose=True, ignore_bg_processes=True)
+        report = "perf report --input=%s" % self.output_file
+        self.run_test(report)
+        if not os.stat(self.output_file).st_size:
+            self.fail("%s sample not captured" % self.output_file)
+
+    def address_v1(self):
+        # Get memory address of single variable
+        output = self.run_test('./dawr_v1')
+        data = output.stdout.decode("utf-8")
+        return data
+
+    def address_v2(self):
+        # Get memory address of two variables
+        output = self.run_test('./dawr_v2')
+        data = output.stdout.decode("utf-8").split(',')
+        return data
+
+    def test_read_dawr_v1_gdb(self):
         """
         Setting Read/Write watchpoint on single variable using awatch and
         executing the program
         """
         child, return_value = self.run_cmd('dawr_v1')
-        i = 0
-        if self.distro_name == "fedora":
-            child.sendline('set debuginfod enabled on')
-            child.expect_exact([pexpect.TIMEOUT, ''])
```

Comment:
Same. This will revert code added via trace: Add Fedora Support to trace testing. Can you rebase?